client/http-client/Cargo.toml

```diff
@@ -34,8 +37,8 @@ tokio = { version = "1.16", features = ["net", "rt-multi-thread", "macros"] }
 
 [features]
 default = ["native-tls"]
-native-tls = ["hyper-rustls/native-tokio", "__tls"]
-webpki-tls = ["hyper-rustls/webpki-tokio", "__tls"]
+native-tls = ["hyper-rustls/native-tokio", "hyper-rustls/ring", "__tls"]
```

Comment:
let's discuss whether we should support both crypto backends.

Rustls now supports both ring and aws-lc-rs but migrating to aws-lc-rs requires more dependencies such as cmake, NASM etc.

Reply:
I'm happy with not exposing it initially, and then as a non-breaking change we can always expose it later or whatever if there's demand :)